app/components/Form/ProjectQuarterlyReportForm.tsx

```diff
@@ -0,0 +1,299 @@
+        .map(({ node }) => node)
+        .filter((report) => report.operation !== "ARCHIVE");
+
+    filteredReports.sort(
+      (a, b) =>
+        a.newFormData.reportingRequirementIndex -
+        b.newFormData.reportingRequirementIndex
+    );
+    const nextIndex =
+      filteredReports.length > 0
+        ? filteredReports[filteredReports.length - 1].newFormData
+            .reportingRequirementIndex + 1
+        : 1;
+
+    return [filteredReports, nextIndex];
+  }, [projectRevision.projectQuarterlyReportFormChanges]);
+
+  return (
+    <div>
+      <header>
+        <h2>Quarterly Reports</h2>
+        <SavingIndicator isSaved={!isUpdating && !isAdding} />
+      </header>
+
+      <div>Quarterly reports status here</div>
+
+      <FormBorder>
+        <Button
+          variant="secondary"
+          onClick={() => addQuarterlyReport(nextQuarterlyReportIndex)}
+          className="addButton"
+        >
+          <FontAwesomeIcon icon={faPlusCircle} /> Add another quarterly report
+        </Button>
+
+        {sortedQuarterlyReports.map((quarterlyReport, index) => {
+          return (
+            <div key={quarterlyReport.id} className="reportContainer">
+              <header className="reportHeader">
+                <h3>Quarterly Report {index + 1}</h3>
+                <Button
+                  variant="secondary"
+                  size="small"
+                  onClick={() =>
+                    deleteQuarterlyReport(
+                      quarterlyReport.id,
+                      quarterlyReport.operation
+                    )
+                  }
+                >
+                  Remove
+                </Button>
+              </header>
+              <FormBase
+                id={`form-${quarterlyReport.id}`}
+                idPrefix={`form-${quarterlyReport.id}`}
+                ref={(el) => (formRefs.current[quarterlyReport.id] = el)}
+                formData={quarterlyReport.newFormData}
+                onChange={(change) => {
+                  updateFormChange(
+                    { ...quarterlyReport, changeStatus: "pending" },
+                    change.formData
+                  );
+                }}
+                schema={projectReportingRequirementSchema as JSONSchema7}
+                uiSchema={quarterlyReportUiSchema}
+                ObjectFieldTemplate={EmptyObjectFieldTemplate}
+              />
+            </div>
+          );
+        })}
+      </FormBorder>
+      <Button
+        size="medium"
+        variant="primary"
+        onClick={stageQuarterlyReportFormChanges}
+        disabled={isUpdating}
+      >
+        Submit Quarterly Reports
+      </Button>
+
+      <style jsx>{`
+        div :global(button.pg-button) {
+          margin-left: 0.4em;
+          margin-right: 0em;
+        }
+        div :global(.right-aligned-column) {
+          display: flex;
+          justify-content: flex-end;
+          align-items: flex-start;
+        }
```

Comment:
is this used?

Reply:
Nope! I'll also remove it in the place I borrowed it from that it isn't used